rustls/src/msgs/message.rs

```diff
@@ -93,6 +93,21 @@ impl OpaqueMessage {
         }
     }
 
+    /// Access the message payload as a slice.
+    pub fn payload(&self) -> &[u8] {
+        &self.payload.0
+    }
+
+    /// Access the message payload as a mutable `Vec<u8>`.
+    ///
+    /// This is useful for decrypting a message in-place.  After the message
+    /// is decrypted, call [`OpaqueMessage::into_plain_message()`] or
+    /// [`OpaqueMessage::into_tls13_unpadded_message()`] (depending on the
+    /// protocol version).
```

Comment:
The context about `into_plain_message()` etc feels out of place here, since this is a fairly straightforward getter...

Reply:
moved up into the main docs for this type